tests/integrate_test/cita_features_test.sh

```diff
@@ -55,6 +62,13 @@ main() {
     echo -n "5) Check permission denied ... "
     test_perm_denied 2
     echo "Done"
+
+    echo -e "6) Run block interval tests ...\n"
+    test_change_block_interval 2
+    timeout=$(check_height_growth_normal 0 60) && (echo "FAILED"
```

Comment:
maybe 10 or 15?

Reply:
I want to keep this timeout the same with line 53

Reply:
That's ok.

Reply:
> I want to keep this timeout the same with line 53

The purpose is not same.